kinetic/distribution.yaml

```diff
@@ -3038,7 +3038,7 @@ repositories:
       tags:
         release: release/kinetic/{package}/{version}
       url: https://github.com/UTNuclearRoboticsPublic/move_base_to_manip-release.git
-      version: 1.0.6-2
+      version: 1.0.9-0
```

Comment:
I think I'm encountering a problem since splitting 'master' into 'indigo' and 'kinetic' branches.' I just ran bloom again, creating tag 1.0.10 for kinetic. But that tag already exists for indigo. Run it again to create tag 1.0.11? (see attached)
![1_0_10_already_exists](https://cloud.githubusercontent.com/assets/11284393/23625203/edf1ab08-026d-11e7-81ba-fe671d4a6379.png)